examples/rfq_quickstart/main.rs

```diff
@@ -183,7 +184,7 @@ async fn main() {
             chain,
             rfq_tokens,
             cli.tvl_threshold,
-            [sell_token_address.clone()].into(),
+            default_quote_tokens_for_chain(chain).expect(""),
```

Comment:
Let's move this into the hashflow client itself so that the user doesn't need to call this manually